docs/ReleaseNotes.md

```diff
@@ -0,0 +1,77 @@
+
+#### Show only rows exceeding threshold
+The **Show only rows exceeding threshold** button allows you to turn on mode in which all rows with **Similarity** value not exceeding **Threshold** are hidden.
+This feature can be combined with the filtering calculated results by the **Threshold** parameter.
+
+#### New Source Origin Modes
+In previous version of Antiplagiarism, you could only use the whole current solution as a source code for the comparison. Now you can choose one of the three modes:
+
+1. **Current Solution** to use the whole current solution for the comparison.
+2. **Selected Project** to use selected project from the current solution. The **Select Project** combobox will appear in which you can choose the project for comparison.
+3. **Selected Folder** to use selected folder with source code for comparison. You can specify the folder in the appeared **Select Folder** panel.
+
+#### New Supported Work Modes
+In previous version of Antiplagiarism, you could only specify the solution file as a reference solution for comparison. Now you can choose one of the three modes:
+
+1) **Self-Analysis** to compare the source code (selected with a consideration for specified **Source Origin**) with itself. This mode is useful for refactoring.
+2) **Reference Solution** to compare the source code with provided reference solution. The reference solution can be selected in the appeared **Reference Solution** control.
+control. You can select solution (.sln) or project (.csproj) file.
+3) **Acumatica Sources** to compare the source code with a code stored in a specified folder. This mode was designed for third-party developers which do not have
+the whole Acumatica solution but have a set of source code files delivered with Acumatica ERP. However, this mode can be used in a more general way not related to Acumatica 
+to compare source code with a code stored in the specified folder. The folder can be selected in the appeared **Acumatica Sources** control.
+
+### Other Enhancements
+* The logging for errors is greatly improved.
+* The integraion with Visual Studio is improved. Added the background loading of the package.
+* If you enter invalid values in the **Threshold** or **Min. Method Size** textbox the textbox's background will become red and the useful tooltip
+about allowed range of values will be shown.
+* Added support for Visual Studio 2019.
+* A lot of small UI enhancements - tooltips, messages, enable/disable UI controls depending on the context, etc.
+
+### Bug Fixes
+In this version, the following bugs have been fixed.
+
+| Bug | Fix Description |
+| --- | --------------- |
+| The sort didn't work for the **Similarity** column. | The sort now works for all data grid columns. |
+| The data from the **Similarity** column wasn't copied to the clipboard. | The data now is copied to the clipboard for all data grid columns. |
+
+
+## Antiplagiarism 1.0
```

Comment:
Why do we have version 1.3 and 1.0? Where is 1.1 and 1.2? :)

Reply:
@k-s-popova we decided that there were quite big changes in the project. A lot of things were improved - the code analysis algorithm, UI, etc. The 1.1 version just doesn't indicate the quality of changes :)
Also we are not the first ones - where is windows 9 or iphone 9? It's a common marketing trend:)